lib/database.cpp

```diff
@@ -19,11 +19,11 @@
 using namespace Quotient;
 Database::Database()
 {
-    QSqlDatabase::addDatabase(QStringLiteral("QSQLITE"));
+    QSqlDatabase::addDatabase(QStringLiteral("QSQLITE"), QStringLiteral("Quotient"));
     QString databasePath = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
     QDir(databasePath).mkpath(databasePath);
-    QSqlDatabase::database().setDatabaseName(databasePath + QStringLiteral("/database.db3"));
-    QSqlDatabase::database().open();
+    QSqlDatabase::database(QStringLiteral("Quotient")).setDatabaseName(databasePath + QStringLiteral("/quotient.db3"));
+    QSqlDatabase::database(QStringLiteral("Quotient")).open();
```

Comment:
Ugh - I really expected that you wrap this in an accessor :) Or at least, please make a static constant for `QStringLiteral("Quotient")`.

Reply:
and i fully expected this comment :)